packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/RulesTestsUtils.java

```diff
@@ -48,9 +48,9 @@
      *
      * @return The created rule model.
      */
-    public static RestRuleModel createRuleModelWithDefaultValues()
+    public static RestRuleModel createRuleModelWithModifiedValues()
```

Comment:
I'm not getting the name change, why it contains "ModifiedValues" when the method creates a rule with predefined default values?

Reply:
The idea is that if you create a rule without specifying any of the new "other" values, they all have some value. 
The method name above was confusing (at least to me) cos it modifies the default values of the fields, but it was called `defaultValues`.

Reply:
Ohh ok, now I understand - I'm ok with that.